apps/ledger-live-mobile/src/components/HeaderTitle.tsx

```diff
@@ -7,7 +7,10 @@ import { scrollToTop } from "../navigation/utils";
 import { Merge } from "../types/helpers";
 
 export default function HeaderTitle(
-  props: Merge<HeaderTitleProps, { children?: React.ReactNode }>,
+  props: Merge<
+    HeaderTitleProps,
+    { children?: React.ReactNode; color?: BaseTextProps["color"] }
```

Comment:
Is there a better way to include `BaseTextProps`?

Reply:
We could try to use `Pick<BaseTextProps, "color">`